pkg/scaffold/v2/manager/config.go

```diff
@@ -88,6 +86,11 @@ spec:
       containers:
       - command:
         - /manager
+        env:
```

Comment:
shouldn't be necessary -- the leader election code determines this automatically, IIRC